[2022-07-28 13:57:45,881] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: glue_af_pipeline.create_table manual__2022-07-28T13:57:44.699671+00:00 [queued]>
[2022-07-28 13:57:45,895] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: glue_af_pipeline.create_table manual__2022-07-28T13:57:44.699671+00:00 [queued]>
[2022-07-28 13:57:45,897] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 13:57:45,899] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-28 13:57:45,901] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 13:57:45,914] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): create_table> on 2022-07-28 13:57:44.699671+00:00
[2022-07-28 13:57:45,921] {standard_task_runner.py:52} INFO - Started process 1319 to run task
[2022-07-28 13:57:45,925] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'glue_af_pipeline', 'create_table', 'manual__2022-07-28T13:57:44.699671+00:00', '--job-id', '157', '--raw', '--subdir', 'DAGS_FOLDER/loaddata_rds.py', '--cfg-path', '/tmp/tmpi4b72dol', '--error-file', '/tmp/tmpo6yakn7s']
[2022-07-28 13:57:45,929] {standard_task_runner.py:80} INFO - Job 157: Subtask create_table
[2022-07-28 13:57:45,987] {task_command.py:371} INFO - Running <TaskInstance: glue_af_pipeline.create_table manual__2022-07-28T13:57:44.699671+00:00 [running]> on host dd3800c43a8f
[2022-07-28 13:57:46,098] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=glue_af_pipeline
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-07-28T13:57:44.699671+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-28T13:57:44.699671+00:00
[2022-07-28 13:57:46,109] {base.py:68} INFO - Using connection ID 'psql_aws' for task execution.
[2022-07-28 13:57:46,340] {dbapi.py:231} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                ccustomerID TEXT NOT NULL,
                gender TEXT NOT NULL,
                SeniorCitizen TEXT NOT NULL,
                Partner TEXT NOT NULL,
                Dependents TEXT NOT NULL,
                tenure TEXT NOT NULL,
                PhoneService TEXT NOT NULL,
                MultipleLines TEXT NOT NULL,
                InternetService TEXT NOT NULL,
                OnlineSecurity TEXT NOT NULL,
                OnlineBackup TEXT NOT NULL,
                DeviceProtection TEXT NOT NULL,
                TechSupport TEXT NOT NULL,
                StreamingTV TEXT NOT NULL,
                StreamingMovies TEXT NOT NULL,
                Contract TEXT NOT NULL,
                PaperlessBilling TEXT NOT NULL,
                PaymentMethod TEXT NOT NULL,
                MonthlyCharges TEXT NOT NULL,
                TotalCharges TEXT NOT NULL,
                Churn TEXT NOT NULL)
        , parameters: None
[2022-07-28 13:57:46,415] {postgres.py:94} INFO - NOTICE:  relation "users" already exists, skipping

[2022-07-28 13:57:46,434] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=glue_af_pipeline, task_id=create_table, execution_date=20220728T135744, start_date=20220728T135745, end_date=20220728T135746
[2022-07-28 13:57:46,502] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-28 13:57:46,551] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
